package/collection2/collection2.js

```diff
@@ -476,7 +474,7 @@ function doValidate(collection, type, args, getAutoValues, userId, isFromTrusted
 
     return args;
   } else {
-    error = getErrorObject(validationContext, `in ${collection._name} ${type}`);
+    error = getErrorObject(validationContext, Meteor.isProduction ? '' : `in ${collection._name} ${type}`);
```

Comment:
I understand that you made this change for https://github.com/Meteor-Community-Packages/meteor-collection2/issues/422 but if we're going to implement this it's better to let users decided for themselves instead of forcefully adding with no way to change it.

Reply:
How about adding `disableCollectionNamesInProduction` property to collection2 and having it on by default?

Reply:
Will do, but we will set this on for the next major release.